_schemaV2.graphql

```diff
@@ -7263,6 +7266,21 @@ type Me implements Node {
   unreadNotificationsCount: Int!
 }
 
+# Collection of fields that describe medium type
+type MediumType {
+  # A globally unique ID.
+  id: ID!
+
+  # A type-specific ID likely used as a database ID.
+  internalID: ID!
+
+  # Long descriptive phrase used as companion for short_description
+  longDescription: String
+
+  # Shortest form of attribution class display
```

Comment:
```suggestion
  # Shortest form of medium type display
```

Reply:
Whoops this suggestion is in the wrong place (generated schema). Should be [over in mediumType.ts](https://github.com/artsy/metaphysics/pull/2898/files#diff-fc80c52d8ab1c792e1e19717981c0020cc8d06b435d74687e142b22c8d2712a2R12)